recipes/treelite/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "treelite" %}
+{% set version = "0.92" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmlc/treelite/archive/{{ version }}.tar.gz
+  sha256: 17961a4f4c5933630dfee5c7129180e9d69660f3efa05f87bc8b168081bf3fd7
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake >=3.14
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - python
+    - pip >=19.0
+    - libprotobuf >=3.10
```

Please remove the constraint so the global pinning will be applied.
```suggestion
    - libprotobuf
```